<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Post2dWindowArrowStructuredSettingDialog</class>
 <widget class="QDialog" name="Post2dWindowArrowStructuredSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arrow Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="physValLabel">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Physical value:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="solutionComboBox"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="lengthGroupBox">
         <property name="title">
          <string>Length</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="lengthAutoCheckBox">
            <property name="text">
             <string>Auto</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="stdValueLabel">
                <property name="text">
                 <string>Standard value:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="stdValueSpinBox">
                <property name="decimals">
                 <number>6</number>
                </property>
                <property name="minimum">
                 <double>0.001000000000000</double>
                </property>
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="stdValueUnitLabel">
                <property name="text">
                 <string>[m/s]</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="legendLenLabel">
                <property name="text">
                 <string>Length on screen:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="legendLengthSpinBox">
                <property name="minimum">
                 <number>10</number>
                </property>
                <property name="maximum">
                 <number>1000</number>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="legendLenUnitLabel">
                <property name="text">
                 <string>[pixel]</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="minValueLabel">
                <property name="text">
                 <string>Minimum value to draw:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="minValueSpinBox">
                <property name="decimals">
                 <number>8</number>
                </property>
                <property name="minimum">
                 <double>0.001000000000000</double>
                </property>
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QLabel" name="minValueUnitLabel">
                <property name="text">
                 <string>[m/s]</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="samplingGroupBox">
         <property name="title">
          <string>Sampling</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="samplingAllRadioButton">
            <property name="text">
             <string>All vertices</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="samplingSkipRadioButton">
            <property name="text">
             <string>Sampling rate:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="iSamplingRateLabel">
              <property name="text">
               <string>I-direction</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="iSamplingRateSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>99</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="jSamplingRateLabel">
              <property name="text">
               <string>J-direction</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="jSamplingRateSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>99</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="colorGroupBox">
         <property name="title">
          <string>Color</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QRadioButton" name="specificRadioButton">
            <property name="text">
             <string>Custom color</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="ColorEditWidget" name="colorEditWidget"/>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="scalarComboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="scalarRadioButton">
            <property name="text">
             <string>By scalar value</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Arrow Shape</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="arrowSizeLabel">
              <property name="text">
               <string>Arrow size:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="arrowSizeSpinBox">
              <property name="minimum">
               <number>3</number>
              </property>
              <property name="maximum">
               <number>20</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lineWidthLabel">
              <property name="text">
               <string>Line width:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="lineWidthSpinBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>11</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>157</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="legendTextGroupBox">
         <property name="title">
          <string>Length Legend</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="vtkTextPropertySettingWidget" name="legendTextWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="regionSettingButton">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&amp;Region Setting</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="colorbarSettingButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&amp;Color Bar Setting</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorEditWidget</class>
   <extends>QLabel</extends>
   <header>guibase/widget/coloreditwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>vtkTextPropertySettingWidget</class>
   <extends>QWidget</extends>
   <header>guibase/vtktextpropertysettingwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Post2dWindowArrowStructuredSettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Post2dWindowArrowStructuredSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lengthAutoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>stdValueSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lengthAutoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>legendLengthSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>samplingSkipRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>iSamplingRateSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>samplingSkipRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>jSamplingRateSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>specificRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorEditWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scalarRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>scalarComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lengthAutoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>minValueSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scalarRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorbarSettingButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
